From patchwork Fri Apr 17 00:12:31 2020
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Fabio Estevam <festevam@gmail.com>
X-Patchwork-Id: 1271952
Return-Path: <u-boot-bounces@lists.denx.de>
X-Original-To: incoming@patchwork.ozlabs.org
Delivered-To: patchwork-incoming@bilbo.ozlabs.org
Authentication-Results: ozlabs.org;
 spf=pass (sender SPF authorized) smtp.mailfrom=lists.denx.de
 (client-ip=2a01:238:438b:c500:173d:9f52:ddab:ee01; helo=phobos.denx.de;
 envelope-from=u-boot-bounces@lists.denx.de; receiver=<UNKNOWN>)
Authentication-Results: ozlabs.org;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: ozlabs.org; dkim=pass (2048-bit key;
 unprotected) header.d=gmail.com header.i=@gmail.com header.a=rsa-sha256
 header.s=20161025 header.b=dm2uGgsI; dkim-atps=neutral
Received: from phobos.denx.de (phobos.denx.de
 [IPv6:2a01:238:438b:c500:173d:9f52:ddab:ee01])
 (using TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits)
 key-exchange X25519 server-signature RSA-PSS (4096 bits))
 (No client certificate requested)
 by ozlabs.org (Postfix) with ESMTPS id 493GlG38TMz9sP7
 for <incoming@patchwork.ozlabs.org>; Fri, 17 Apr 2020 10:13:14 +1000 (AEST)
Received: from h2850616.stratoserver.net (localhost [IPv6:::1])
 by phobos.denx.de (Postfix) with ESMTP id 7790B81A42;
 Fri, 17 Apr 2020 02:12:51 +0200 (CEST)
Authentication-Results: phobos.denx.de;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: phobos.denx.de;
 spf=pass smtp.mailfrom=u-boot-bounces@lists.denx.de
Authentication-Results: phobos.denx.de; dkim=pass (2048-bit key;
 unprotected) header.d=gmail.com header.i=@gmail.com header.b="dm2uGgsI";
 dkim-atps=neutral
Received: by phobos.denx.de (Postfix, from userid 109)
 id EBA5381A26; Fri, 17 Apr 2020 02:12:28 +0200 (CEST)
X-Spam-Checker-Version: SpamAssassin 3.4.2 (2018-09-13) on phobos.denx.de
X-Spam-Level: 
X-Spam-Status: No, score=-2.0 required=5.0 tests=BAYES_00,DKIM_SIGNED,
 DKIM_VALID,DKIM_VALID_AU,FREEMAIL_FROM,SPF_HELO_NONE autolearn=ham
 autolearn_force=no version=3.4.2
Received: from mail-qv1-xf42.google.com (mail-qv1-xf42.google.com
 [IPv6:2607:f8b0:4864:20::f42])
 (using TLSv1.3 with cipher TLS_AES_128_GCM_SHA256 (128/128 bits))
 (No client certificate requested)
 by phobos.denx.de (Postfix) with ESMTPS id 15CA481A0D
 for <u-boot@lists.denx.de>; Fri, 17 Apr 2020 02:12:15 +0200 (CEST)
Authentication-Results: phobos.denx.de;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: phobos.denx.de;
 spf=pass smtp.mailfrom=festevam@gmail.com
Received: by mail-qv1-xf42.google.com with SMTP id c12so86431qvj.5
 for <u-boot@lists.denx.de>; Thu, 16 Apr 2020 17:12:15 -0700 (PDT)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=gmail.com; s=20161025; 
 h=from:to:cc:subject:date:message-id:in-reply-to:references;
 bh=Mf6zowPBPwhIPvcJI/BgRu9N+Y6fcSrVS+IvrYnBHrc=;
 b=dm2uGgsIoOF5C93wDHPuzTxeI1jsZ6A4h2uF8wJwcCCBxZFK4L2hFrAUcikk3Bv3i3
 LRhPuhUX4SgbRmVFe7jq0UWQ7FriymmBIFYuHPz1swrZXUpmUf0Sd8KFvM5WF3GI/9CT
 zHg8VGiq6GeOgKmAGA1+Xe6wIo5AaxbCss+Q/aCYtQVEBBioRZYKbAvTqzRz5zr+cPG/
 T4fak+lL883bZsk5e73DvQJ/KMf0DFvhTZsh94yITu3Kn1PlBB2soNAFlhn6IauDcAne
 3Xxs7Vi50ZUMBy4eo0xJMEV9n7k6nsckRqGVQ5wLR8T1VxSk7xtCjoefPFzkscrIsgAn
 nODA==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
 d=1e100.net; s=20161025;
 h=x-gm-message-state:from:to:cc:subject:date:message-id:in-reply-to
 :references;
 bh=Mf6zowPBPwhIPvcJI/BgRu9N+Y6fcSrVS+IvrYnBHrc=;
 b=N1ObkuDnOjryLTlpfkVJUXgzlrHGZVroHLniNfE8ouO54/vYjW28caS6kNm0dwkfwb
 hgqe1CBeosPCWc45RJHeypo3L834kLoUHdhk3nQIdjbuRfkWYNqG7c5FscvnAIsEtdQm
 xOZEMk6TUJESxVKZ8k00pnrXhlBsGg7eCZIV7hhWeNxU4f3pbdf5hykzaWDRNEVdd5qS
 5cBtIIak99yXXcMrvdA/mFiNu2XKtC8jB/FEyX+T1q+k6XagQdDwpZBTJ7ruH7+JKLWl
 0std625wGDomSiLvuRBJopYvteyjve4WPmb0OKk6zJ84nOhww8EFhopoJU8aGyQeMKnA
 FIUw==
X-Gm-Message-State: AGi0Pub2usCHtWlwkYviWpm1I2Lqx/1s4ao9GJ/ieyJbjjdQa7u+fdKV
 zngUBDTfjyovPiuJfuh8v7s=
X-Google-Smtp-Source: APiQypIIsglFSJ1m6yE+gpbNQ60jQC9FzDtFokX/ZPaz+cjxbkGYx/vgu8xC3KWQtBAYc6uP0xpa3Q==
X-Received: by 2002:a0c:ca0e:: with SMTP id c14mr270609qvk.134.1587082333890; 
 Thu, 16 Apr 2020 17:12:13 -0700 (PDT)
Received: from localhost.localdomain ([2804:14c:482:271:809b:81f0:1d41:9d6b])
 by smtp.gmail.com with ESMTPSA id
 x5sm16987996qtm.81.2020.04.16.17.12.11
 (version=TLS1_3 cipher=TLS_AES_256_GCM_SHA384 bits=256/256);
 Thu, 16 Apr 2020 17:12:13 -0700 (PDT)
From: Fabio Estevam <festevam@gmail.com>
To: sbabic@denx.de
Cc: hs@denx.de, derek@ihtfp.com, u-boot@lists.denx.de,
 Fabio Estevam <festevam@gmail.com>
Subject: [PATCH 5/5] wandboard: Remove checkboard()
Date: Thu, 16 Apr 2020 21:12:31 -0300
Message-Id: <20200417001231.11406-5-festevam@gmail.com>
X-Mailer: git-send-email 2.17.1
In-Reply-To: <20200417001231.11406-1-festevam@gmail.com>
References: <20200417001231.11406-1-festevam@gmail.com>
X-BeenThere: u-boot@lists.denx.de
X-Mailman-Version: 2.1.30rc1
Precedence: list
List-Id: U-Boot discussion <u-boot.lists.denx.de>
List-Unsubscribe: <https://lists.denx.de/options/u-boot>,
 <mailto:u-boot-request@lists.denx.de?subject=unsubscribe>
List-Archive: <https://lists.denx.de/pipermail/u-boot/>
List-Post: <mailto:u-boot@lists.denx.de>
List-Help: <mailto:u-boot-request@lists.denx.de?subject=help>
List-Subscribe: <https://lists.denx.de/listinfo/u-boot>,
 <mailto:u-boot-request@lists.denx.de?subject=subscribe>
Errors-To: u-boot-bounces@lists.denx.de
Sender: "U-Boot" <u-boot-bounces@lists.denx.de>
X-Virus-Scanned: clamav-milter 0.102.2 at phobos.denx.de
X-Virus-Status: Clean

Since the variant revision is no longer show, simply get rid
of checkboard().

Move the gpio_request() prior to its usage.

Signed-off-by: Fabio Estevam <festevam@gmail.com>
---
 board/wandboard/wandboard.c | 15 +--------------
 1 file changed, 1 insertion(+), 14 deletions(-)

diff --git a/board/wandboard/wandboard.c b/board/wandboard/wandboard.c
index df454d8f17..bb1ffd87e1 100644
--- a/board/wandboard/wandboard.c
+++ b/board/wandboard/wandboard.c
@@ -404,6 +404,7 @@ static const struct boot_mode board_boot_modes[] = {
 static bool is_revc1(void)
 {
 	SETUP_IOMUX_PADS(rev_detection_pad);
+	gpio_request(REV_DETECTION, "REV_DETECT");
 	gpio_direction_input(REV_DETECTION);
 
 	if (gpio_get_value(REV_DETECTION))
@@ -466,20 +467,6 @@ int board_init(void)
 	return 0;
 }
 
-int checkboard(void)
-{
-	gpio_request(REV_DETECTION, "REV_DETECT");
-
-	if (is_revd1())
-		puts("Board: Wandboard rev D1\n");
-	else if (is_revc1())
-		puts("Board: Wandboard rev C1\n");
-	else
-		puts("Board: Wandboard rev B1\n");
-
-	return 0;
-}
-
 #ifdef CONFIG_SPL_LOAD_FIT
 int board_fit_config_name_match(const char *name)
 {
